DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data-Driven Decisions\n",
+    "\n",
+    "In this piece, we'll get down to the core of what data means, how we can extract knowledge from information from data, and how to use it as a savvy businessperson, researcher, or analyst. You won't need any preparation, but there will be some code available for the aspiring programmers to use and modify. You will walk away from this page with a better understanding of what your data needs or needs not and how to most effectively to use it.\n",
+    "\n",
+    "\n",
+    "\n",
+    "# Part 1: Information vs Data\n",
+    "\n",
+    "Webster's online dictionary defines Data as factual measurements or statistics used as a basis for actions such as reasoning, deliberations, or calculation. We want to pay specific attention to the word \"basis\" here, as it highlights the fact that data can be, quite easily, manipulated. If we don't want to be susceptible to this manipulation, then we should make sure we turn that data into information. We see that Information's entry is \"knowledge obtained from investigation, study, or instruction,\" and that is really what's at play when we're making smart, data-driven decisions:\n",
+    "\n",
+    "## Fallacies and Manipulation\n",
+    "\n",
+    "<!-- cite: https://www.datasciencecentral.com/profiles/blogs/data-fallacies-to-avoid-an-illustrated-collection-of-mistakes -->\n",
+    "\n",
+    "The first step in being a conscious member of the data community is to identify and refute bad data appropriately. There are many resources for the many, varied ways to manipulate data, but we'll look at only the most common and most relevant ones here.\n",
```

Comment:
- Reword second sentence to "there are many resources for the various ways data is manipulated..."